fixtures/book2.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
Thank you for the observation. I will remove it. I was using it to test my api for multiple file upload.